frontend/src/pages/server/components/message/MessageComponent.tsx

```diff
@@ -1,44 +1,81 @@
-import React, { FunctionComponent } from 'react'
+import React, { FunctionComponent, useRef } from 'react'
 
 interface MessageComponentProps {
     content?: string
     timestamp?: object
+    media?: string
     createdBy: {
         username?: string
         photoUrl?: string
     }
+    onMessageLoaded: Function
 }
 
 const MessageComponent: FunctionComponent<MessageComponentProps> = ({
     content,
     timestamp,
     createdBy,
+    media,
+    onMessageLoaded,
 }) => {
+    const imageRef = useRef<HTMLImageElement>(null)
+
     // Firebase save timestamp as object, but still number when console log
     // So we need to manually cast to number
     const serverTime = timestamp as unknown as number
 
     // Get time from timestamp
-    const messageTime = new Date(serverTime).toLocaleString([], {
-        hour: '2-digit',
-        minute: '2-digit',
-    })
+    const getMessageTime = () =>
```

Comment:
should be utils function
    const getMessageTime = (time) => new Date(time);
codes don't relate to ComponentView should be in the outside